input/vocabulary/valueset-encounter-classHistory-class.json

```diff
@@ -0,0 +1,123 @@
+    },
+    "extension": [
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+            "valueCode": "pa"
+        },
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+            "valueInteger": 1
+        }
+    ],
+    "url": "http://terminology.hl7.org/5.0.0/ValueSet-v3-ActEncounterCode.html",
```

Comment:
This is the Official URL:  http://terminology.hl7.org/ValueSet/v3-ActEncounterCode

Reply:
Updated the Official URL.